packages/map-template/src/components/Sidebar/Modal/Modal.jsx

```diff
@@ -41,7 +44,15 @@ function Modal({ children, isOpen }) {
         }
     }, [contentRef]);
 
-    return <div ref={modalRef} className={`modal ${isOpen ? 'modal--open' : ''} ${fullHeight ? 'modal--full' : ''} ${substeps ? 'modal--substeps' : ''}`}>
+    return <div ref={modalRef}
+        className={`modal ${isOpen ? 'modal--open' : ''} ${fullHeight ? 'modal--full' : ''} ${substeps ? 'modal--substeps' : ''}`}
+        style={{
```

Comment:
Could we add a BEM modifier instead?

Reply:
That's a good idea, I added a BEM modifier instead.

Reply:
🙌